### ISSUE TITLE
Handling unknown values

### ISSUE THREAD

**Issue by Airy59** [StructLab/CCS-TMS-1.0#2]
The TTL file seems to reflect the JSON schema practice to use fictitious values as placeholders for unknown values.

Here is an example:

###  http://data.europa.eu/949/reversalTime
era:reversalTime rdf:type owl:DatatypeProperty ;
                 rdfs:domain [ rdf:type owl:Class ;
                               owl:unionOf ( era:Derailer
                                             era:Switch
                                           )
                             ] ;
                 rdfs:range xsd:unsignedInt ;
                 era:ccsView "INFRA" ;
                 era:exp "-1"^^xsd:int ;
                 era:unitOfMeasure unit:SEC ;
                 dcterms:created "2023-11-13"^^xsd:date ;
                 rdfs:comment "Defines the time in seconds to switch from one position to the other ; 0 means unknown"@en ;
                 rdfs:label "Reversal Time"@en .

Two major issues:

**1 - not adhering to ontology design principles**

Such practice is not in line with the very principles of ontologies, namely the Open World Assumption : what is unknown is just unknown. If there were a fact stating :this_switch :reversalTime "0"ˆˆxsd:int, it states the fact that reversal time is zero. The actual fact is that reversal time is not known.

If and when the fact is known at some point in time, the fact will be added. Since ontologies rely on description logic, which is monotonic, the new fact will not invalidate previously made inferences.

**2 - acceptable in a SIL4 environment?**

Using a conventional value for unknown is:

- extremely prone to misunderstanding, hence not suitable for a model aimed at being widely shared
- extremely prone to miscalculations, when for instance evaluating the maximum reversal time from a collection of reversal time (see slip switches). If there is a "zero-means_unknown", then the result of Max(rt1, rt2, ...) will be a lower estimate, not the true Max.
In other words, by redefining what zero means, you are also customizing the meaning of "addition".

Note: the issue was already raised months ago in Polarion, inter alia.
